src/org/opendatakit/briefcase/export/ExportForms.java

```diff
@@ -60,8 +61,14 @@ public static ExportForms load(ExportConfiguration defaultConfiguration, List<Fo
     forms.forEach(form -> {
       String formId = getFormId(form);
       ExportConfiguration load = ExportConfiguration.load(exportPreferences, buildCustomConfPrefix(formId));
-      if (!load.isEmpty())
+      if (((BriefcaseFormDefinition) form.getFormDefinition()).isFileEncryptedForm() ||
+              ((BriefcaseFormDefinition) form.getFormDefinition()).isFieldEncryptedForm()) {
+        //Check if this is an encrypted form. If true then set config to know it needs a PEM file
+        load.setFormNeedsPrivateKey(Boolean.TRUE);
```

Comment:
Think that the only reason to change this value would be if a conf changes its form and this doesn't make sense. This value needs to be set on creation and is never to be changed (make it `final`, by the way)

Reply:
Yeah agree that this should be made final. Had a lot of trouble finding all the possible scenarios where users could change config

Reply:
Thanks for your review will update the PR